tracks/julia/exercises/pythagorean-triplet/mentoring.md

```diff
@@ -5,14 +5,15 @@
 
 ## Example solutions
 
-A O(n^2) time algorithm:
-
 ````
+A O(n²) time algorithm:
```

Comment:
Since you started here, do you feel like search/replace-ing all ^2 in comments with `²`? :D